### PR TITLE
Setup dev release 2 dev1

### PR DESCRIPTION
**Pull request by markowsiak**
Setting up a development release of v2.0.0-dev1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,7 +102,8 @@
     "socks-proxy-agent": "^5.0.0",
     "update-notifier": "4.1.0",
     "uuid": "3.4.0",
-    "xml2js": "^0.4.23"
+    "xml2js": "^0.4.23",
+    "keytar": "5.4.0"
```
Comment (jblz) [inline package.json]:
```suggestion
    "xml2js": "^0.4.23"
```

This change seems unrelated (unless I've missed something)

**Review by jblz — COMMENTED** [review]
Looks good other than the stray keytar package inclusion.

File: package.json (hunk 1 above)
Comment (markowsiak) [reply package.json]: Yeah I'm not sure why that's in there tbh, I'll check it out.

File: .publishrc [hunk 2]
```diff
@@ -5,11 +5,11 @@
     "untrackedFiles": true,
     "sensitiveData": true,
     "branch": "master",
-    "gitTag": true
+    "gitTag": false
   },
   "confirm": true,
   "publishCommand": "npm publish",
-  "publishTag": "beta",
+  "publishTag": "next",
```
Comment (jblz) [inline .publishrc:12]: `Check that git tag matches version specified in the package.json`

Wouldn't we still want this?

Comment (markowsiak) [reply .publishrc:12]: Just following what is in the CONTRIBUTING.md.  `next` for .publishrc and `-dev1` in the version property.
